src/test/kotlin/no/nav/syfo/personstatus/api/v2/DialogmotekandidatPersonoversiktStatusApiV2Spek.kt

```diff
@@ -92,10 +60,61 @@ object DialogmotekandidatPersonoversiktStatusApiV2Spek : Spek({
                     }
                 }
 
+                it("return NoContent for a person with a tilfelle, who is kandidat, but it's historic") {
+                    createPersonoversiktStatusWithTilfelle(database)
+                    setAsKandidat(database, OffsetDateTime.now().minusDays(365))
```

Comment:
Du kan vurdere å bruke named parameter på disse, så slipper man å sjekke metoden for å vite hva datoen gjelder 🤔